atom/common/api/resources/windows_bindings.js

```diff
@@ -17,7 +17,21 @@ binding.registerCustomHook(function (bindingsAPI, extensionId) {
   })
 
   apiFunctions.setHandleRequest('get', function (windowId, getInfo, cb) {
-    console.warn('chrome.windows.get is not supported yet')
+    if (arguments.length === 1) {
```

Comment:
you actually don't need all of these checks because`setHandleRequest` normalizes the arguments